lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -843,9 +859,15 @@ namespace Js
         {
             OUTPUT_TRACE_DEBUGONLY(Js::ModulePhase, _u("\t>NotifyParentsAsNeeded (errorObject)\n"));
             NotifyParentsAsNeeded();
-            return;
+            return false;
         }
 
+        return true;
+    }
+
+    void SourceTextModuleRecord::GenerateRootFunction()
+    {
+        ScriptContext* scriptContext = GetScriptContext();
```

Comment:
Make sense to assert this->WasDeclarationInitialized() ?

Reply:
Sounds good, done!